lib/v2/gov/shanghai/wgj/wgj.js

```diff
@@ -0,0 +1,75 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const queryString = require('query-string');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+module.exports = async (ctx) => {
+    const baseUrl = 'http://wsbs.wgj.sh.gov.cn';
+    const currentUrl = `${baseUrl}/shwgj_ywtb/core/web/welcome/index!toResultNotice.action?flag=1`;
+    const page = ctx.params.page ?? 1;
+    const params = {
+        'pageDoc.pageNo': page,
+    };
+    const response = await got({
+        method: 'post',
+        url: currentUrl,
+        headers: {
+            'Content-Type': 'application/x-www-form-urlencoded',
+        },
+        searchParams: queryString.stringify(params),
```

Comment:
You can pass [an object to searchParams](https://github.com/sindresorhus/got/tree/v11.8.6#searchparams) without the need of query-string
```suggestion
        searchParams: {
            flag: 1,
            'pageDoc.pageNo': page,
        },
```